.github/workflows/ci-core.yml

```diff
@@ -53,9 +54,31 @@ jobs:
         id: ignore-filter
         run: echo "changes=true" >> $GITHUB_OUTPUT
 
+      - name: Filter other changes
+        uses: ./.github/actions/filter-changes
+        id: filter
+        with:
+          # Filter files that match **every** pattern in this list
+          every-filter: |
+            - '**'                     # Initially include all files
+            - '!**/*.md'               # Exclude all markdown files (lowercase)
+            - '!**/*.MD'               # Exclude all markdown files (uppercase)
```

Comment:
This will not work. We embed markdown files in our go source.

Reply:
@jmank88 I’ve made the necessary adjustments in this commit: https://github.com/smartcontractkit/chainlink/pull/14585/commits/f4b1f98b9284fbc00ce9958728b3842bd5c4a780. Now, unit tests will run even when there are changes to markdown files. Tests will be skipped only when changes are made to integration-tests/**, .github/** actions, or workflows, unless those changes are related to the ci-core.yml workflow itself.